src/intl/en/page-upgrades-vision.json

```diff
@@ -21,34 +23,34 @@
   "page-upgrades-vision-security": "Security",
   "page-upgrades-vision-security-desc": "The planned upgrades improve Ethereum's security against coordinated attacks, like a 51% attack. This is a type of attack where if someone controls the majority of the network they can force through fraudulent changes.",
   "page-upgrades-vision-security-desc-3": "The transition to proof-of-stake means that the Ethereum protocol has greater disincentives against attack. This is because in proof-of-stake, the validators who secure the network must stake significant amounts of ETH into the protocol. If they try and attack the network, the protocol can automatically destroy their ETH.",
   "page-upgrades-vision-security-desc-5": "This isn't possible in proof-of-work, where the best a protocol can do is force entities who secure the network (the miners) to lose mining rewards they would have otherwise earned. To achieve the equivalent effect in proof-of-work, the protocol would have to be able to destroy all of a miner's equipment if they try and cheat.",
   "page-upgrades-vision-security-desc-5-link": "More on proof of work",
-  "page-upgrades-vision-security-desc-8": "Ethereum's security model also needs to change because of the introduction of shard chains. The Beacon Chain will randomly assign validators to different shards - this makes it virtually impossible for validators to ever collude by attacking a specific shard. Sharding isn't as secure on a proof-of-work blockchain, because miners can't be controlled by the protocol in this way.",
-  "page-upgrades-vision-security-desc-10": "Staking also means you don't need to invest in elite hardware to 'run' an Ethereum node. This should encourage more people to become a validator, increasing the network's decentralization and decreasing the attack surface area.",
+  "page-upgrades-vision-security-desc-8": "Ethereum's security model also needs to change because of the introduction of sharding. The Beacon Chain will coordinate all of the validators who will be responsible for asserting that all data has been made available, but every node will no longer be required to hold the entire history of the chain. A new role is also anticipated, known as a dedicated block builder, who will work alongside block proposers (validators) for efficient and safe block production. Proof-of-stake is a prerequisite to sharding.",
+  "page-upgrades-vision-security-desc-10": "Staking also means you don't need to invest in elite hardware to participate directly in consensus. This should encourage more people to become a validator, increasing the network's decentralization and decreasing the attack surface area.",
   "page-upgrades-vision-security-staking": "Stake ETH",
-  "page-upgrades-vision-security-validator": "You can become a validator by staking your ETH.",
+  "page-upgrades-vision-security-validator": "You can become a validator by staking your ETH, but remember, you don't need any ETH to run your own node.",
```

Comment:
I don't like this addition. The original acts more as a CTA for people to become stakers.

Reply:
```suggestion
  "page-upgrades-vision-security-validator": "You can become a validator by staking your ETH.",
```